src/plug-rc2-docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,235 @@
+version: '3'
+services:
+  # pumba:
+  #   container_name: pumba
+  #   image: alpine-pumba #image can be built with pumba/Dockerfile: docker build -t alpine-pumba .
+  #   user: "root"
+  #   volumes:
+  #     - /var/run/docker.sock:/var/run/docker.sock
+  #   networks:
+  #     tx-network:
+  #       ipv4_address: 10.6.0.20
+  #     block-network:
+  #       ipv4_address: 10.5.0.20        
+  
+  tx-node-1:
```

Comment:
Naming: We want a set of folders all with consistent names.
The naming convention I would like to use is:
* folders = [blockchain]-[version]
 eg. plug-1.0.0-rc1, cennznet-1.0.0-rc2
* nodes = node-[blockchain]-[version]-n
 eg. node-plug-1_0_0-rc2-2, boot-node-plug-1_0_0-rc1-3
This ensures that docker containers do not get mixed up and we have a consistent way of adding new blockchain options